producer/ue_authentication.go

```diff
@@ -222,7 +221,6 @@ func UeAuthPostRequestProcedure(updateAuthenticationInfo models.AuthenticationIn
 
 		var eapPkt radius.EapPacket
 		var randIdentifier int
-		rand.Seed(time.Now().Unix())
```

Comment:
Why is this deleted? Deleting this, means that the seed for `randIdentifier` will always be the same (default `seed = 1`), correct?
The error I see in my system is "error file=producer/ue_authentication.go,line=225,col=3::SA1019: rand.Seed has been deprecated since Go 1.20 and an alternative has been available since Go 1.0: As of Go 1.20 there is no reason to call Seed with a random value. Programs that call Seed with a known value to get a specific sequence of results should use New(NewSource(seed)) to obtain a local random generator." Please see my next comment for a suggestion.

Reply:
(Please see my answer in the other comment). I'll resolve the comment here to keep the conversation in 1 place.